lib/features/identity_creator/data/model/identity_cache_model.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:jmap_dart_client/jmap/identities/identity.dart';
+import 'package:json_annotation/json_annotation.dart';
+import 'package:tmail_ui_user/features/identity_creator/domain/model/identity_cache.dart';
+import 'package:tmail_ui_user/features/manage_account/presentation/model/identity_action_type.dart';
+import 'package:tmail_ui_user/features/public_asset/data/model/public_assets_in_identity_arguments_model.dart';
+
+part 'identity_cache_model.g.dart';
+
+@JsonSerializable(includeIfNull: false, explicitToJson: true)
+class IdentityCacheModel extends IdentityCache {
```

Comment:
maybe should have more clearly in the name of this model. what it use for?

Reply:
It is for caching identity content.